README.md

```diff
@@ -2,10 +2,221 @@
+2. Open the `CMakeLists.txt` found in `mp\src`
+3. In the top section, you may see "x64-Debug"/"x64-Release" as a selected profile. This needs to be changed to "x86-Debug"/"x86-Release" as this is a 32-bit project. To do so...
+4. Click on the dropdown, go to: "Manage Configurations..."
+5. Click the green plus button and select "x86-Debug" for debug or "x86-Release" for release mode and apply the configuration.
+6. Then make sure to change it over to "x86-Debug" or "x86-Release".
+7. In the "Solution Explorer", it'll be under the "Folder View". To switch to the cmake view, right-click and click on "Switch to CMake Targets View".
+
+After that, it should be able to compile. For debugger/run cmake configuration, refer to: [CONTRIBUTING.md - Debugging - VS2022 + cmake (Windows)](CONTRIBUTING.md#vs2022--cmake-windows).
+
+#### Qt Creator 6+ (Linux)
+1. On the "Welcome" screen, click on "Open Project..."
+2. Open the `CMakeLists.txt` found in `mp/src`
+3. It may ask about kit configuration, tick both Debug and Release configuration and set their build directories ending in "...build/debug" and "...build/release" respectively.
+4. By default, the build is not done in parallel but rather sequentiality. Note, parallel builds at the default setting could deadlock the system or make it unresponsive during the process. Available since cmake 3.12, the amount of jobs can be tweaked using `--parallel <jobs>` where `<jobs>` is a number to specify parallel build level, or just simply don't apply it to turn it off. To turn on parallel builds in Qt Creator: On the "Projects" screen, in [YOUR KIT (under Build & Run)] > Build, go to "Build Steps" section, expand by clicking on "Details", and add `--parallel` to the CMake arguments.
+
+After that, it should be able to compile. For debugger/running configuration, refer to: [CONTRIBUTING.md - Debugging - Qt Creator 6+ (Linux)](CONTRIBUTING.md#qt-creator-6-linux)
+
+#### CLI (with ninja, Windows + Linux)
+##### Windows prerequisite
+Make sure the "x86 Native Tools Command Prompt for VS2022" is used instead of the default.
+
+##### Linux prerequisite - Steam Runtime 3 "Sniper" Container
+Just download and use the OCI image for Docker/Podman/Toolbx:
+* [Docker](https://www.docker.com/): `sudo docker pull registry.gitlab.steamos.cloud/steamrt/sniper/sdk`
+* [Podman](https://podman.io/): `podman pull registry.gitlab.steamos.cloud/steamrt/sniper/sdk`
+* [Toolbx](https://containertoolbx.org/): `toolbox create -i registry.gitlab.steamos.cloud/steamrt/sniper/sdk sniper; toolbox enter sniper`
+
+Depending on the terminal, you may need to install an additional terminfo in the container just to make it usable.
+The container is based on Debian 11 "bullseye", just look for and install the relevant terminfo package if needed.
+
+##### CLI Building steps
+Using with the ninja build system, to build NT;RE using the CLI can be done with:
+
+```
+$ cd /PATH_TO_REPO/neo/mp/src
```

Comment:
Missing step to run docker container (only for [Docker] and [Podman]?)
For [Docker] it would be:
```
docker run -v /PATH_TO_REPO/neo/mp/src:/root/neo/mp/src --rm -it --entrypoint /bin/bash registry.gitlab.steamos.cloud/steamrt/sniper/sdk
cd /root/neo/mp/src/
```

Reply:
I think podman maps practically 1:1 to docker, just without need of special privilage. Nevertheless tried your example with podman and it worked so.